views/tables/ChartsTable/index.jsx

```diff
@@ -137,7 +137,7 @@ const ChartsTable = ({
           </Link>
         )
       case 'updatedAt':
-        return chart[property] ? dayjs(chart[property]).format(DATE_FORMAT) : ''
+        return dayjs(chart[property]).format(DATE_FORMAT)
```

Comment:
won't it still be possible to have old data without this property?

Reply:
We haven't migrated charts yet, so I don't see this being an issue

Reply:
it'll be an issue on all charts in staging, no?

Reply:
<img width="1440" alt="Screenshot 2024-03-22 at 9 41 14 AM" src="https://github.com/AMP-SCZ/dpdash/assets/19805355/ef6d67b1-145a-4a9d-acdd-69759e01a874">
looks fine to me